### PR TITLE
bump libc to 0.2.95

### PR DESCRIPTION
**Pull request by brotzeit**

### CODE REVIEW

File: build-aux/config.guess [hunk 1]
```diff
@@ -1483,8 +1482,8 @@ EOF
     i*86:rdos:*:*)
 	echo "$UNAME_MACHINE"-pc-rdos
 	exit ;;
-    i*86:AROS:*:*)
-	echo "$UNAME_MACHINE"-pc-aros
+    *:AROS:*:*)
+	echo "$UNAME_MACHINE"-unknown-aros
```
Comment (DavidDeSimone) [inline build-aux/config.guess:1486]: Why did we need to change this? Did it produce a new build error once we bumped libc?

**Comment by harryfei** [comment]
is "0.2" enough to get latest version of crate?

File: build-aux/config.guess (hunk 1 above)
Comment (brotzeit) [reply build-aux/config.guess:1486]: This is not related to this pr, but this always changes when I build emacs-ng. Do these files not change when you run the build ?

**Comment by brotzeit** [comment]
> is "0.2" enough to get latest version of crate?

No idea. But explicit seems more reasonable ?

File: build-aux/config.guess (hunk 1 above)
Comment (brotzeit) [reply build-aux/config.guess]: I dropped the changes.

**Review by harryfei — APPROVED** [review]

**Comment by DavidDeSimone** [comment]
I missed your response to my comment, but this looks like a good PR.
Glad it was merged.

**Pull request merged**